docs/docusaurus.config.js

```diff
@@ -81,13 +81,21 @@ const config = {
             title: 'Docs',
             items: [
               {
-                label: 'User Guide',
+                label: 'Draft User Guide',
                 to: '/docs/user-guide',
               },
               {
-                label: 'Specification',
+                label: 'Current Javadoc',
+                to: '/docs/api/org/jspecify/nullness/package-summary.html',
+              },
+              {
+                label: 'Draft Spec',
                 to: '/docs/spec',
               },
+              {
+                label: 'JSpecify Wiki',
```

Comment:
```suggestion
                label: 'Wiki',
```

otherwise "JSpecify" goes everywhere.